scripts/generate-changelog.js

```diff
@@ -214,7 +214,7 @@ class Formatter {
       text += textChange + (count > 1 ? ` (${count} times). ` : '. ');
     };
 
-    return text;
+    return text.trimEnd();
```

Comment:
This requires Node.js v8 or higher: https://v8.dev/features/string-trimming#support I wasn't sure which Node.js version this repo is meant to support, but if it's an older version, we could use `trimRight` instead.